docker/compose/swarm/scale_segmentstore

```diff
@@ -24,7 +24,10 @@ try:
     zk_url = get_config("ZK_URL")
 
     existing = subprocess.check_output("docker service ls -f 'name=pravega_segmentstore_'", shell=True)
+    segments_version = subprocess.check_output("docker service ls -f 'name=pravega_segmentstore'", shell=True)
```

Comment:
the above command and this one only has "_" different in ` 'name=pravega_segmentstore'"`
why cant we use the "existing variable and extract info from it rather than calling the docker command again?

Reply:
I intend to compare with the original segment store (as that is configured using the yaml file) . The '_' ensures that only the second segment store and higher are extracted.

Reply:
Also the existing call would return no output when the scale up is done for the first time i.e the number of segment store is 1.

Reply:
As per the documentation ref: https://docs.docker.com/engine/reference/commandline/service_ls/ we can use multiple filter options, this can be used to reduce the number of number of docker calls.
(To further optimize it we should verify if wild cards are supported ...)

Reply:
Will look into this & test with multiple filters to reduce docker calls. Thank you for pointing this out.